### ISSUE TITLE
[B16] Control Seal: Use - GUI can't be opened when attached to Empty Air

### ISSUE THREAD

**Issue by MalkContent** [Azanor/thaumcraft-beta#1045]
Since this can get worked around easily this is just an inconvenience. Also I have a suspicion that fixing this requires annoying special handling. Just thought I'd put this on the tracker for the record.

**Comment by Azanor** [comment]
Indeed, this requires a lot of special coding. I will keep it open in case I get around to it at some point.
